lsif-semanticdb/src/main/java/com/sourcegraph/lsif_semanticdb/LsifSemanticdb.java

```diff
@@ -112,9 +114,21 @@ private Integer processDocument(LsifTextDocument doc, Set<String> isExportedSymb
         }
 
         // Hover
+        ArrayList<MarkedString> markedStrings = new ArrayList<>();
         String documentation = symbolInformation.getDocumentation().getMessage();
         if (!documentation.isEmpty()) {
-          int hoverId = writer.emitHoverResult(doc.semanticdb.getLanguage(), documentation);
+          markedStrings.add(new MarkedString(Semanticdb.Language.UNKNOWN_LANGUAGE, documentation));
```

Comment:
Why do we use unknown language here?

Reply:
Im a bit unsure about how to approach this one. 

- On one side, by setting it to unknown language, we can extract the proper pretty printed signature for the SnapshotLsifCommand (explained [here](https://github.com/sourcegraph/lsif-java/pull/131/files/d82dd45e3c3f20d429b4c7b1f47d66824b7aa449#diff-94acd677e36ffbda144a22796bd114db1af4154de5513b67a5241a83a02df453R142)
- On the other hand, we may lose syntax highlighting of the javadoc comment. This one's a bit of a double edged sword, as depending on the highlighter, it may end up highlighting words that are not meant to be (like `this` when used in a sentence as opposed to a code snippet in the javadoc comment)

Reply:
That sounds good for a follow-up PR after discussing in our 1:1 :+1: